### ISSUE TITLE
Screenshot is not working in chrome when test cases executed 2nd time 

### ISSUE THREAD

**Issue by uday-nitjsr** [cypress-io/cypress#1820]
### Is this a Feature or Bug?
Bug

### Current behavior:
Screenshot is not working in chrome 2nd time when the test case are executed via GUI mode 

```
Object has been destroyed
    at BrowserWindow.capturePage (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\electron.asar\browser\api\browser-window.js:203:16)
    at C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\gui\windows.js:141:24
    at Object.takeScreenshot (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\gui\windows.js:140:18)
    at invoke (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\browsers\electron.js:148:31)
    at onRequest (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\browsers\electron.js:166:28)
    at C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\automation\index.js:78:20
    at attempt (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\screenshots.js:98:14)
    at captureAndCheck (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\screenshots.js:116:7)
    at Object.capture (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\screenshots.js:277:14)
    at Object.capture (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\automation\screenshot.js:11:28)
    at C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\automation\index.js:88:33
    at normalize (C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\automation\index.js:85:30)
    at C:\Users\UdayN\AppData\Local\Cypress\Cache\3.0.0\Cypress\resources\app\packages\server\lib\automation\index.js:125:20
    at runCallback (timers.js:781:20)
    at tryOnImmediate (timers.js:743:5)
    at processImmediate [as _immediateCallback] (timers.js:714:5)
From previous event:
    at $Cypress.automation (http://localhost:4100/__cypress/runner/cypress_runner.js:62423:12)
    at automate (http://localhost:4100/__cypress/runner/cypress_runner.js:59690:20)
    at automateScreenshot (http://localhost:4100/__cypress/runner/cypress_runner.js:59696:12)
    at scrollAndTake (http://localhost:4100/__cypress/runner/cypress_runner.js:59744:12)
From previous event:
    at takeScrollingScreenshots (http://localhost:4100/__cypress/runner/cypress_runner.js:59746:18)
    at takeFullPageScreenshot (http://localhost:4100/__cypress/runner/cypress_runner.js:59773:10)
    at http://localhost:4100/__cypress/runner/cypress_runner.js:59918:16
From previous event:
    at takeScreenshot (http://localhost:4100/__cypress/runner/cypress_runner.js:59911:19)
    at Context.screenshot (http://localhost:4100/__cypress/runner/cypress_runner.js:60010:14)
    at Context.<anonymous> (http://localhost:4100/__cypress/runner/cypress_runner.js:63860:21)
    at http://localhost:4100/__cypress/runner/cypress_runner.js:63575:33
From previous event:
    at runCommand (http://localhost:4100/__cypress/runner/cypress_runner.js:63565:14)
    at next (http://localhost:4100/__cypress/runner/cypress_runner.js:63647:14)
    at next (http://localhost:4100/__cypress/runner/cypress_runner.js:63635:16)
    at http://localhost:4100/__cypress/runner/cypress_runner.js:63659:18
From previous event:
    at next (http://localhost:4100/__cypress/runner/cypress_runner.js:63647:34)
From previous event:
    at http://localhost:4100/__cypress/runner/cypress_runner.js:63666:37
From previous event:
    at run (http://localhost:4100/__cypress/runner/cypress_runner.js:63664:15)
    at Object.cy.(anonymous function) [as request] (http://localhost:4100/__cypress/runner/cypress_runner.js:63884:11)
    at Context.runnable.fn (http://localhost:4100/__cypress/runner/cypress_runner.js:64024:20)
    at callFn (http://localhost:4100/__cypress/runner/cypress_runner.js:33042:21)
    at Test.Runnable.run (http://localhost:4100/__cypress/runner/cypress_runner.js:33035:7)
    at http://localhost:4100/__cypress/runner/cypress_runner.js:67001:28
From previous event:
    at Object.onRunnableRun (http://localhost:4100/__cypress/runner/cypress_runner.js:66996:20)
    at $Cypress.action (http://localhost:4100/__cypress/runner/cypress_runner.js:62309:51)
    at Test.Runnable.run (http://localhost:4100/__cypress/runner/cypress_runner.js:66067:20)
    at Runner.runTest (http://localhost:4100/__cypress/runner/cypress_runner.js:33505:10)
    at http://localhost:4100/__cypress/runner/cypress_runner.js:33611:12
    at next (http://localhost:4100/__cypress/runner/cypress_runner.js:33425:14)
    at http://localhost:4100/__cypress/runner/cypress_runner.js:33435:7
    at next (http://localhost:4100/__cypress/runner/cypress_runner.js:33367:14)
    at http://localhost:4100/__cypress/runner/cypress_runner.js:33403:5
    at timeslice (http://localhost:4100/__cypress/runner/cypress_runner.js:28644:27)
logError @ cypress_runner.js:140547
```
### Desired behavior:
Screenshot should be taken
### Steps to reproduce:

1. add cy.screenshot() in a test
2. open cypress GUI mode
3. Execute test case on chrome (works fine)
4. Close browser and execute same test on Electron
5. Execute test cases on chrome again(fails now with the provided stacktrace)
below is sample code for test case
```js
it("verify articles on Global feeds",function(){
  var allArticles
  cy.request("http://localhost:3000/api/articles?limit=10&offset=0")
  .then((response)=>{
    allArticles = response.body.articles		
  })

  cy.visit("/")

  cy.get("ul.nav-pills a.nav-link")
  .contains("Global Feed")
  .click()
  .should("have.class","active")

  cy.screenshot()

  cy.get("div.article-preview")
  .should("be.visible")
  .each(function($ele,index){
    cy.wrap($ele)
    .find("a.preview-link>h1")
    .should("be.visible")
    .should("have.text",allArticles[index].title)
    
    cy.wrap($ele)
    .find("a.preview-link")
    .should("have.attr","href","/article/"+allArticles[index].slug)

    cy.wrap($ele)
    .find("a")
  })
})
```

### Versions
Cypress: 3.0.0
OS: Windows 7

**Comment by chrisbreiding** [comment]
Thanks for the bug report.

This is a state issue in the server related to using electron and then switching to any other browser. We'll get this fixed ASAP, but in the meantime, restart Cypress and as long as you avoid switching from electron to another browser, screenshots should work properly.

**Comment by jennifer-shehane** [comment]
Released in [`3.0.1`](https://docs.cypress.io/guides/references/changelog.html#3-0-1).
